content/blog/cookie-consent-banner.mdx

```diff
@@ -0,0 +1,102 @@
+---
+title: Cookie Consent Banner
+date: 2024-05-08
+image: /assets/blog/ccb/preview.png
+hero: /assets/blog/ccb/hero.png
+descriptionShort: The Cookie Consent Banner is one of the very first FOSS projects published by Porsche back in 2021. In this Contributor Story, the two maintainers Tim and Patrick share what they have learnt over the past three years and how they managed to keep an eye on the needs of the community.
+author:
+  name: Cookie Consent Banner
+  description: Simplify cookie consent with Cookie Consent Banner, providing a customizable UI and a straightforward API. Focus on the things that really matter.
+  image: https://raw.githubusercontent.com/porscheofficial/cookie-consent-banner/main/assets/logo.svg
+  slug: https://github.com/porscheofficial/cookie-consent-banner
+readTime: 5min
+---
+
+<ImageText imageSrc="/assets/blog/ccb/3.png" imageAlt="The Open Source Maintainers, co-working on a laptop" theme="light"
+    title="Tell us a little more about yourself, your role at Porsche and what topics drive you.">
+
+    **Tim (Web Developer Customer Innovation / Physical Products):** I'm passionate
+    about enabling best-in-class user experiences for our products. To achieve
+    this, we all rely on software packages that must meet high quality
+    standards. It is therefore particularly important to me to be an active
+    member of the open-source world and to contribute our ambitious standards
+    back to the community.
+    <br/>
+    **Patrick (System Architect My Porsche):** I'm a huge fan of a great user
+    experience. Not only for the end users of a product, but also for the software
+    engineers creating those. An awesome developer experience leads to an awesome
+    (end-) user experience. That's why I enjoy spending time into defining a
+    meaningful and scalable interface definition. The Cookie Consent Banner is such
+    an example. It's a component that you have to build for legal reasons and
+    probably don't touch quite often after the initial implementation. It shouldn't
+    be complicated if you need to adapt it again after some time. The Cookie Consent
+    Banner is very flexible, but still convenient to integrate.  
+
+</ImageText>
+
+<ImageText imageSrc="/assets/blog/ccb/1.png" imageAlt="Collection of websites using the cookie banner" theme="light" left
+    title="You are currently maintaining Porsche's open-source project Cookie Consent Banner on GitHub. How would you explain your project to a five-year-old? What problem does your project solve for Porsche?">
```

Comment:
The answer won't be understandable for a five-year-old. Maybe rephrase the question to `in simple terms`?

Reply:
I rephrased it to "in a nutshell"